src/insns/packw.adoc

```diff
@@ -0,0 +1,49 @@
+[#insns-packw,reftext="Pack low 16-bits of registers (RV64)"]
```

Comment:
This seems to be the only instruction that uses the convention of putting "(RV64)" here if it's not supported on RV32. Should all the other W instructions do the same or should this be dropped for this instruction?